apiary.apib

```diff
@@ -1883,6 +1883,8 @@ Get details for a single invoice.
                     + contact (object)
                         + type: `contact` (string)
                         + id: `f88ab83c-b8ca-497b-a2c6-1ad631fe1cea` (string)
+                + email: `duivels@test.com` (string, nullable)
```

Comment:
Q: Should we name this to something more explicit? Which email is represented here?

Reply:
Are you suggesting something like `invoicing_email`? @stivni @jenssegers ?

Reply:
@mrtus having it under `invoicee` I would assume that this is the email in which the invoice will be sent, is that correct?

Reply:
Since it is inside the invoicee, I think it is exclicit it is the invoicee/customer email

Reply:
Discussed this with @PieterReel and we decided that this is specific enough, indeed